packages/dds/map/src/directory.ts

```diff
@@ -192,24 +192,45 @@ export type IDirectoryOperation = IDirectoryStorageOperation | IDirectorySubDire
 
 /**
  * Defines the in-memory object structure to be used for the conversion to/from serialized.
- * @privateRemarks
- * Directly used in JSON.stringify, direct result from JSON.parse.
+ *
+ * @remarks Directly used in
+ * {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify
+ * | JSON.stringify}, direct result from
+ * {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse | JSON.parse}.
  */
 export interface IDirectoryDataObject {
+    /**
+     * TODO
+     */
     storage?: { [key: string]: ISerializableValue; };
+
+    /**
+     * TODO
+     */
     subdirectories?: { [subdirName: string]: IDirectoryDataObject; };
 }
 
+/**
+ * {@link IDirectory} storage format.
+ */
 export interface IDirectoryNewStorageFormat {
+    /**
+     * TODO
```

Comment:
What are the semantics of this? Do we have a common definition for blobs that we could reference here?

Reply:
Not really sure on this one offhand.

Reply:
@vladsud Do you have insight into this?